### PR TITLE
zboss: version v3.11.2.0+v5.1.2

### PR DESCRIPTION
**Pull request by FilipZajdel**

### CODE REVIEW

File: zboss/development/src/CMakeLists.txt [hunk 1]
```diff
@@ -13,12 +13,6 @@ zephyr_library_include_directories(
     include
   )
 
-# Prevent the -Waddress-of-packed-member warning. We appear to be
-# taking addresses of struct fields in a manner that is not
-# safe. Until this is resolved upstream, we temporarily remove this
-# warning.
-zephyr_library_compile_options(-Wno-address-of-packed-member)
-
```
Comment (tomchy) [inline zboss/development/src/CMakeLists.txt:21]: Please bring it back :)

File: zboss/production/src/CMakeLists.txt [hunk 2]
```diff
@@ -13,12 +13,6 @@ zephyr_library_include_directories(
     include
   )
 
-# Prevent the -Waddress-of-packed-member warning. We appear to be
```
Comment (tomchy) [inline zboss/production/src/CMakeLists.txt:16]: Please bring it back too :)

**Review by tomchy — CHANGES_REQUESTED** [review]

**Review by tomchy — APPROVED** [review]

File: zboss/CHANGELOG.rst [hunk 3]
```diff
@@ -18,11 +18,30 @@ Added
...
+* New ZBOSS API addons, allowing to declare ZCL8-compatible devices.
+
+Updated
+=======
+
+* Updated the ZBOSS stack to version ``3.11.2.0+5.1.2``.
+  For detailed information, see `ZBOSS stack release notes`_ for the main branch.
+* Updated the development ZBOSS stack to version ``3.12.1.0+5.2.0``.
+  For detailed information, see `ZBOSS stack release notes`_ for the main branch.
+* Updated ZCL in all variants of ZBOSS to specification version ``8``.
```
Comment (greg-fer) [inline zboss/CHANGELOG.rst:30]:
```suggestion
* Updated ZCL in all variants of ZBOSS to specification version 8.
```

File: zboss/CHANGELOG.rst [hunk 4]
```diff
@@ -18,11 +18,30 @@ Added
...
+  This version is backward-compatible with earlier versions of ZCL.
+* Improved the NCP communication stability.
 
 Removed
 =======
 
 * Precompiled libraries for experimental Zigbee Green Power Proxy Basic functionality for End Device role.
+* Legacy HA device definitions due to ZCL8 incompatiblities.
+  The sample device definitions can be found inside the ``include`` directory in every Zigbee sample and application.
```
Comment (greg-fer) [inline zboss/CHANGELOG.rst:39]:
```suggestion
  The sample device definitions can be found inside the :file:`include` directory in every Zigbee sample and application.
```

File: zboss/CHANGELOG.rst [hunk 5]
```diff
@@ -18,11 +18,30 @@ Added
...
+
+Bug fixes
+=========
+
+* Fixed an issue where incorrect Read Attributes Response is sent on reading multiple attributes when the first attribute is unsupported.
```
Comment (greg-fer) [inline zboss/CHANGELOG.rst:44]:
```suggestion
* Fixed an issue where incorrect Read Attributes Response would be sent on reading multiple attributes when the first attribute is unsupported.
```

**Review by greg-fer — APPROVED** [review]
Approving, but please contact me so that we can coordinate uploading the docs with the merge of this PR.

**Pull request merged**